wesnoth-manual.po

```diff
@@ -353,25 +353,18 @@ msgstr "Načíst hru"
 
 #. type: Content of: <book><chapter><section><variablelist><varlistentry><listitem><simpara>
 #: doc/manual/manual.en.xml:153
-#, fuzzy
-#| msgid ""
-#| "Click this button to load a previously saved game. You will be shown a "
-#| "dialogue listing saved games. Select the game and click Ok to load and "
-#| "continue, or Cancel to return to the Main Menu. If you select a replay "
-#| "game, you can check the Replay check box. The loaded game will make all "
-#| "the moves from the beginning while you watch."
 msgid ""
 "Click this button to load a previously saved game. You will be shown a "
 "dialogue listing saved games. Select the game and click Ok to load and "
 "continue, or Cancel to return to the Main Menu. If you select a replay game, "
 "you can check the \"Show replay\" check box. The loaded game will make all "
 "the moves from the beginning while you watch."
 msgstr ""
-"Tímto tlačítkem můžete načíst uložené hry. Zmáčknutím se dostanete do "
-"dialogu, kde lze vybrat hru ze seznamu; Ok ji načte, Zrušit Vás vrátí do "
-"hlavního menu. Pokud vyberete místo hry záznam, můžete zaškrtnout „Ukaž "
-"záznamy“ - po načtení si budete moci přehrát záznam celé hry tak, jak byla "
-"původně odehrána."
+"Kliknutím na toto tlačítko lze načíst dříve uloženou hru. Zobrazí se dialog "
+"se seznamem uložených her. Výběrem hry a kliknutím na Ok ji načteš, "
+"tlačítkem Zrušit se vrátíš do hlavního menu. Pokud vybereš místo hry záznam, "
+"můžeš zaškrtnout políčko „Ukaž záznamy“. Načtený záznam hry ti přehraje "
+"všechny tahy od začátku hry."
```

Comment:
Ten první přeložený odstavec je perfektní, ale tady jsem narazil. Tak jsem hledal na webu, spustil hru a také prohledával .po soubory. To vše proto, aby se manuál příliš nevzdálil hře.

K tomu mám ještě vůči vám mindrák, protože nemám prozkoumáno, jak manuál funguje. Tedy ani to, jak moc je udržovaný vůči verzím 1.14 a master (to doufám poznám, až budu vaše změny cpát do 1.14), ale ani to, kdy se dostane na místo určení - [na webu](https://wiki.wesnoth.org/WesnothManual) píšou, že sem a také, že je přibalen ke hře. Když jsem to naposled zkoumal, tak změny, které jsme do té doby provedli, se na webu neprojevily. Snad až bude 100 %? Nehledal jsem.

Navíc z manuálu čiší kopie Nápovědy ve hře, což mi nepřipadá jako systémové řešení. Tedy viděl bych za dobré, kdyby existoval jediný zdroj informací, který se překládá, a z něj se automaticky vytvářela nápověda, manuál, web... Ale to pláču na špatném hrobě. Píšu to jen proto, že já, kdybych se k tomu dostal, tak bych při překladu ještě ověřoval, jak ladí s nápovědou ve hře.

[dialog](https://en.wikipedia.org/wiki/Dialog_box) = [dialogové okno](https://cs.wikipedia.org/wiki/Dialogov%C3%A9_okno)

**Ok** není ani v 1.14 ani v master

[záznam](https://wiki.wesnoth.org/Replays) je vlastnost uložené hry

**Show replay** je v 1.14 i master přeloženo jako **Přehrát záznam** - tohle by chtělo změnit i ve hře, protože jako **Přehrát záznam** jsem přeložil **Play Replay**, takže bych **Show replay** přeložil jako **Ukázat záznam**; zatím tedy navrhuji opravu na nepřesný překlad, aby to bylo alespoň stejné
```suggestion
"Kliknutím na toto tlačítko lze načíst dříve uloženou hru. Zobrazí se dialogové okno "
"se seznamem uložených her. Výběrem hry a kliknutím na Načíst hru ji načteš, "
"tlačítkem Zrušit se vrátíš do hlavního menu. Pokud vybereš hru se záznamem, "
"můžeš zaškrtnout políčko „Přehrát záznam“. Načtená hra ti přehraje "
"všechny tahy od začátku, zatímco se budeš dívat."
```

Reply:
Některé věci jsem si neověřil, takže jsem to asi trochu zkomplikoval já. Držel jsem se moc textu a své paměti.
Jinak souhlasím, že je opravdu překombinované, když existují domény help, manual i manpages. Nutí se to ptát, proč tak komplikovaně.

Reply:
Na to bych měl odpověď - komplikované to je, protože je to jednodušší. Tedy udělat každou věc zvlášť (nápovědu ve hře, manuálové stránky v systému).

Abychom to my překladatelé měli jednodušší, museli by se programátoři, nebo spíš analytici, víc snažit při návrhu i implementaci takové univerzální dokumentace.